src/Servers/IIS/build/native.targets

```diff
@@ -5,7 +5,11 @@
   </PropertyGroup>
 
   <ItemGroup Condition="@(MessageFile->Count()) != 0">
-    <CustomBuild Include="@(MessageFile)">
+    <!-- Checking if the IntDir exists is a local development optimization. Normally, each time the build runs,
+      the aspnetcore_msg.h file is auto generated. The compiler notices that the file has been updated, and recompiles c++ files.
+      These files never update (they define message types, which aren't changed). Therefore, if the IntDir exists, don't recreate
+      the header file. -->
+    <CustomBuild Include="@(MessageFile)" Condition="!Exists('$(IntDir)MSG00001.bin')">
```

Comment:
Below our "Outputs" says `MSG0409` is `MSG00001` correct?

Reply:
Yeah, I believe the MSG0409 is some copy pasta from a sample on how to get message compiler to work (https://stackoverflow.com/questions/3026855/configure-the-message-compiler-mc-exe-as-a-custom-compiler-step-in-vc-2010). See https://docs.microsoft.com/en-us/windows/desktop/wes/message-compiler--mc-exe-#remarks. I'll try changing the name and see what breaks.